### DISCUSSION TITLE
More efficient way to sync multiple speakers

### DISCUSSION THREAD

**Discussion by seancmalloy** [music-assistant/support#2073] [Feature requests and ideas]
I would like see a more efficient way to sync multiple speakers. In music assistant its required to sync speakers one by one, where as LMS you can go to one speaker and have a list of all your other speakers you can select with check boxes to sync in one place. This is far easy than MA. If I am wrong can someone explain how I can select multiple speakers in one place.  This may seem like a small thing, but it is enough to keep me with LMS, even though MA  is superior in a lot of ways; especially when mysqueezebox servers are about to be discontinued.

![Screenshot (65)](https://github.com/music-assistant/support/assets/23019226/4054e68a-f160-4b3e-abc6-65b5cdb96e68)
 
![Screenshot (67)](https://github.com/music-assistant/support/assets/23019226/d988b606-4d2f-44ff-9fe7-855e5419b87b)

**Comment by OzGav** [comment]
There are multiple ways to group players https://music-assistant.io/ui/#grouping-players

**Comment by seancmalloy** [comment]
Thanks, ill try it out. but sync'n speakers on the fly would be nice. if you are using a group can you turn off a speaker in that group while playing? 

Yes, apparently you can turn off player in group. But I am experiencing strange behavior when trying to play to group, it always shows another speaker with linked speakers (which I didnt link manually) playing at the same time.  

Im using squeezelite

![Screenshot (68)](https://github.com/music-assistant/hass-music-assistant/assets/23019226/dfce5d1f-573f-4607-93dc-69877d8520a2)

**Reply by marcelveldt** [reply]
yes that is a UI glitch, for now you can get rid of it by setting "hide this player in the UI" on your child players in the MA player settings until we fix the issue in a more clever way

**Comment by marcelveldt** [comment]
Like you already found out, the feature you are looking for is creating the syncgroups.

Doing a multi select manual sync is more a nice to have feature for later. Not a priority at this time.

**Comment by seancmalloy** [comment]
Thanks for input, keep up the good work (all devs)

**Comment by OzGav** [comment]
This has been implemented

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
